Bio/AlignIO/MauveIO.py

```diff
@@ -160,12 +160,12 @@ def _write_record(self, record, record_idx=0):
         # of the produced XMFA file.
         if "start" in record.annotations and "end" in record.annotations:
             suffix0 = "/%s-%s" % (
-                str(record.annotations["start"]),
```

Comment:
Not sure about removing this - seems quite likely an end user might have recorded the start and end as integers?

Reply:
The `"%s"` will take care of that.

Reply:
So it will, thanks.